src/System.Private.CoreLib/shared/System/Collections/ObjectModel/Collection.cs

```diff
@@ -69,6 +69,8 @@ public void Add(T item)
             InsertItem(index, item);
         }
 
+        public void AddRange(IEnumerable<T> collection) => InsertItemsRange(items.Count > 0 ? items.Count : 0, collection);
```

Comment:
Why is that check required? if items.Count == 0, we pass 0 anyway, no?

Reply:
You are correct, not sure why I added this as we can just use `items.Count`